db/memtable.cc

```diff
@@ -688,6 +688,7 @@ Status MemTable::Add(SequenceNumber s, ValueType type,
   // Format of an entry is concatenation of:
   //  key_size     : varint32 of internal_key.size()
   //  key bytes    : char[internal_key.size()]
+  //  seq | type   : Fixed64[56bit(seq) | 8bit(type)]
```

Comment:
The "key bytes" component includes this too as it is the last eight bytes in `char[internal_key.size()]`. Changing that line to "user key bytes : char[internal_key.size() - 8]" is one way to avoid the duplication.